extensions/wikia/AdEngine/js/provider/gpt/adSizeMap.js

```diff
@@ -0,0 +1,57 @@
+/*global define*/
+define('ext.wikia.adEngine.provider.gpt.adSizeMap', [
```

Comment:
What do you think about moving this abstract logic to Wikia/ad-engine. It could be used through bridge in app repo

Reply:
sure, makes sense.